portfolio/src/main/java/com/google/sps/servlets/RandomPhraseServlets.java

```diff
@@ -0,0 +1,39 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/random-phrase")
+public class RandomPhraseServlets extends HttpServlet {
+
+    String[] funFacts = {
+        "Goosebumps are meant to ward off predators",
+        "The moon has moonquakes",
+        "The heads on Easter Island have bodies",
+        "Humans are the only animals that blush",
+        "The wood frog can hold its pee for up to eight months"
+    };
+
+  @Override
+  public void doGet(final HttpServletRequest request, final HttpServletResponse response) throws IOException {
+        String json = convertToJson(funFacts);
+        response.setContentType("application/json;");
+        response.getWriter().println(json);
+    }
+
+  private String convertToJson(final String[] funFacts) {
```

Comment:
I admire that you did this by hand! :) Have you checked if it's possible use gson.toJson() and see if that takes care of all the JSON converting for you? The example code has 2 methods: 1 does it by hand, as you've done here, but another uses GSON to take care of it to make things easier: https://github.com/JesusU076/software-product-sprint/blob/main/walkthroughs/week-2-server/examples/server-stats/src/main/java/com/google/sps/servlets/ServerStatsServlet.java

Reply:
Yes, I just selected to do the JSON by hand to remember how it works